src/QMCWaveFunctions/Fermion/DelayedUpdateCUDA.h

```diff
@@ -62,23 +62,23 @@ class DelayedUpdateCUDA
   Matrix<T, CUDAHostAllocator<T>> U, Binv;
   Matrix<T> V;
   //Matrix<T> tempMat; // for debugging only
-  Matrix<T, CUDAAllocator<T>, MemorySpace::CUDA> U_gpu, V_gpu, Binv_gpu, temp_gpu, Ainv_gpu;
+  Matrix<T, CUDAAllocator<T>> U_gpu, V_gpu, Binv_gpu, temp_gpu, Ainv_gpu;
```

Comment:
The style guide indicates we agree on one variable declaration per line. Here it is also inconsistent.

Reply:
Will fix.

Reply:
Fixed.